Kernel/Net/TCP.h

```diff
@@ -36,7 +36,34 @@ class [[gnu::packed]] TCPOptionMSS {
     NetworkOrdered<u16> m_value;
 };
 
-static_assert(sizeof(TCPOptionMSS) == 4);
```

Comment:
was the remove of this `static_assert` a possible mistake or am I missing something?

Reply:
Thanks, fixed. :)